### PR TITLE
Fix empty line induced NPE

### PR DESCRIPTION
**Pull request by remyrd**

### CODE REVIEW

File: chr.clj [hunk 1]
```diff
@@ -23,8 +23,12 @@
   (subs line 15))
 
 (defn- parse-commands [line]
-  (let [rest (second (string/split line #";"))]
-    (first (string/split rest #"\s+"))))
+  (some-> line
```
Comment (remyrd) [inline chr.clj:26]: `some->` exits when at any point the threaded result is `nil`

File: chr.clj [hunk 2]
```diff
@@ -23,8 +23,12 @@
   (subs line 15))
 
 (defn- parse-commands [line]
-  (let [rest (second (string/split line #";"))]
```
Comment (remyrd) [inline chr.clj:26]: avoid naming your binding `rest` as it can cause confusion with `clojure.core/rest`

File: chr.clj [hunk 3]
```diff
@@ -23,8 +23,12 @@
...
-    (first (string/split rest #"\s+"))))
+  (some-> line
+    not-empty
```
Comment (remyrd) [inline chr.clj:27]: if the line is empty, `nil`

File: chr.clj [hunk 4]
```diff
@@ -37,7 +41,7 @@
 (println)
 
 (->> commands
-     (map parse-commands)
+     (keep parse-commands)
```
Comment (remyrd) [inline chr.clj:44]: `keep` is like `map` but drops non truth-y elements

**Comment by remyrd** [comment]
Cool little script :pray: are some adjustments :)

**Comment by ThaddeusJiang** [comment]
@remyrd thank you, I learned a lot of knowledge from your PR.

**Pull request merged**